flocker/testtools/_flaky.py

```diff
@@ -22,10 +37,179 @@ def flaky(jira_key):
 
     # TODO (see FLOC-3281):
     # - allow specifying which exceptions are expected
-    # - retry on expected exceptions
-    #   - parametrize on retry options
     # - provide interesting & parseable logs for flaky tests
 
+    if isinstance(jira_keys, basestring):
```

Comment:
TBH, I was ambivalent about providing this DWIM support at all. I'd rather each parameter have just one type. It's not _that_ much syntactic burden for callers to wrap things in a list.

Changed to `unicode` for now, but all I need is the slightest breath of a suggestion that it'd be OK to always take a sequence and I'll change it & update the call sites.